src/error/sseError.ts

```
@@ -0,0 +1,3 @@
+export class SSEError<T=unknown> extends Error {
```

ESLint

[error] 'T' is defined but never used.

(@typescript-eslint/no-unused-vars)